Jenkinsfile

```diff
@@ -42,7 +42,12 @@ node(workerLabel) {
         } // end docker
       } // end if
       if (scmVars.GIT_LOCAL_BRANCH ==~ /master|develop/) {
-        docker_push_tags = [scmVars.GIT_LOCAL_BRANCH, 'latest']
+        if (scmVars.GIT_LOCAL_BRANCH == "develop")
```

Comment:
You can get rid of the nested `if .. else if .. else` block. Also, nested `else` is never reached (actually, it is. But only if someone messed up with the branches names)